app/presenters/order_detail_presenter.rb

```diff
@@ -18,6 +18,15 @@ def description_as_html
     end.join(" &mdash; ").html_safe
   end
 
+  def description_as_text
+    name = product.to_s
+    if bundle
+      name.prepend("#{bundle} -- ")
+    else
+      name
+    end.html_safe
+  end
```

Comment:
You're right this is not the most beautiful method. I basically copy and pasted this from the application_helper, don't feel super motivated to change it now. 😜